internal/settings/settings.go

```diff
@@ -3,6 +3,9 @@ package settings
 type Settings struct {
 	//lint:ignore SA5008 go-flags makes use of duplicate struct tags
 	Type string `long:"type" env:"HACKENV_TYPE" default:"kali" choice:"kali" choice:"parrot" description:"The VM to control with this command"`
+	//lint:ignore SA5008 go-flags makes use of duplicate struct tags
+	Keymap      string `long:"keymap" env:"HACKENV_KEYMAP" default:"fr" choice:"fr" choice:"us" description:"The keyboard keymap to force"`
+	NoProvision bool   `long:"noprovision" env:"HACKENV_NOPROVISION" description:"Don't provision the VM"`
```

Comment:
Would you be okay with making it a `Provision` flag instead? I think this would make it easier for new adopters of hackenv.